core/src/main/java/com/keenant/tabbed/tablist/TitledTabList.java

```diff
@@ -66,10 +67,6 @@ private void updateHeaderFooter() {
         PacketContainer packet = new PacketContainer(Server.PLAYER_LIST_HEADER_FOOTER);
         packet.getChatComponents().write(0, WrappedChatComponent.fromText(this.header == null ? "" : this.header));
         packet.getChatComponents().write(1, WrappedChatComponent.fromText(this.footer == null ? "" : this.footer));
-        try {
```

Comment:
Was there a reason why these try catches were removed? Sure it shouldn't throw errors but in case it does, this may just cause more problems?

Reply:
yes. I couldn't compile without removing them after i changed the ProtocolLib version